templates/security-series/index.html

```diff
@@ -0,0 +1,449 @@
+      {{
+        image(
+         url="https://assets.ubuntu.com/v1/338124d1-shields-security.svg",
+         alt="",
+         height="80",
+         width="140",
+         hi_def=True,
+         attrs={"class": ""},
+         loading="auto",
+        ) | safe
+      }}
+      </p>
+      <p class="p-heading--4">Best practices for securing open source</p>
+      <p>In this webinar learn about the common security and compliance issues with open source, five best practices the Ubuntu Security Team implements and how you can ensure the security integrity of your Ubuntu systems.</p>
+      <a class="p-link--external" href="https://www.brighttalk.com/webcast/6793/440071">Register for the webinar</a>
+    </div>
+    <div class="col-4">
+      <p>
+        {{
+        image(
+         url="https://assets.ubuntu.com/v1/bf810a68-AppArmor_logo.svg",
+         alt="",
+         height="80",
+         width="90",
+         hi_def=True,
+         attrs={"class": ""},
+         loading="auto",
+        ) | safe
+      }}      
+      </p>
+      <p class="p-heading--4">Securing Linux Machines with AppArmor</p>
+      <p>AppArmor provides a crucial layer of security around applications. By providing the capability to whitelist an application’s permissible actions, AppArmor enables administrators to apply the principle of least privilege to applications.</p>
+      <a class="p-link--external" href="https://www.brighttalk.com/webcast/6793/440491">Register for the webinar</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep" id="management-series">
+  <div class="row u-equal-height">
+    <div class="col-9">
+      <p class="p-heading--5">Team</p>
+      <h2>Management ‘big-picture’ series</h2>
+      <p class="u-sv2">This webinar series is for CSO, CISO, VPs and Senior Security Engineers and management to find solutions to big picture infrastructure security and management problems.</p>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <p>
+        {{
+        image(
+          url="https://assets.ubuntu.com/v1/81201fd0-Secure+cloud.svg",
+          alt="",
+          height="80",
+          width="140",
+          hi_def=True,
+          attrs={"class": ""},
+          loading="auto",
+        ) | safe
+      }}      
+      </p>
+      <p class="p-heading--4">Securing open source across multi-cloud environments</p>
+      <p>Join this webinar to see how you can ensure performant open source in production environments and achieve maximum availability by reducing downtime and providing access to high and critical CVE fixes.</p>
+      <p>Furthermore, learn how Ubuntu helps organisations remain compliant with government and industry standards and regulations, including Common Criteria EAL2 with FIPS 140-2 Level 1 certified crypto modules.</p>
+      <a class="p-link--external" href="https://www.brighttalk.com/webcast/6793/440517">Register for the webinar</a>
+    </div>
+    <div class="col-6">
```

Comment:
you need a bit of space between this and the next row
```suggestion
    <div class="col-6 u-sv3">
```